plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestDynamicTable.java

```diff
@@ -132,7 +132,7 @@ public void testDoubleWithScientificNotation()
         String tableName = "primitive_types_table";
         String query = "SELECT string_col FROM " + tableName + " WHERE double_col = 3.5E5";
         assertThatNullPointerException().isThrownBy(() -> buildFromPql(pinotMetadata, new SchemaTableName("default", query)))
-                .withFailMessage("");
```

Comment:
:)

Reply:
Hm. Interestingly, in CI the exception message is `null`, but when I run this locally it was not.

Reply:
Oh.

I was running it on Java 15+ where the NPE error messages are more "friendly" 🤦